StandaloneMmPkg/Core/StandaloneMmCore.c

```diff
@@ -587,7 +587,7 @@ MmEntryPoint (
       }
     }
   } else {
-    DEBUG ((DEBUG_ERROR, "No valid communication buffer, no Synchronous MMI will be processed\n"));
+    DEBUG ((DEBUG_INFO, "No valid communication buffer, no Synchronous MMI will be processed\n"));

```

Comment:
@LeviYeoReum as per my understanding, Typically when we have pre-known MM buffer it falls under sync mode (caller is invoking MM after populating data in MM buffer, i.e. SW SMI for x86). Async mode is more about an interrupted MM (which may not have valid data in shared buffer obviously because it is involked in async manner -> similar to x86 HW SMI). In this implementation it seems you do have a valid data in the buffer but are chosing to use Async mode. Can you please share your comments on that?